src/main/java/Duke.java

```diff
@@ -1,10 +1,202 @@
+import java.util.Scanner;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
 public class Duke {
+    private static String FILE_PATH = "./data/";
+    private static String FILE_NAME = "history.txt";
+    private Ui ui;
+    private Storage storage;
+    private TaskList taskList;
+
     public static void main(String[] args) {
```

Comment:
main class code may seem quite cluttered with many functions, maybe its possible to move the task related methods in to the TaskList class since all the methods are related to handling the list of task?